src/lib/io/sol-iio.c

```diff
@@ -117,6 +117,8 @@ struct resolve_absolute_path_data {
 #define SAMPLING_FREQUENCY_BUFFER_PATH SYSFS_DEVICES_PATH "/iio:device%d/buffer/sampling_frequency"
 #define SAMPLING_FREQUENCY_TRIGGER_PATH SYSFS_DEVICES_PATH "/trigger%d/sampling_frequency"
 
+#define SYSFS_MOUNT_MATRIX SYSFS_DEVICES_PATH "/iio:device%d/in_mount_matrix"
```

Comment:
Thanks @edersondisouza. FYI, the mount matrix data processing logic is github.com:01org/android-iio-sensors-hal.git.  I will add the "out_mount_matrix" checking later